app/models/post.rb

```diff
@@ -0,0 +1,13 @@
+class Post < ApplicationRecord
+  belongs_to :author, class_name: 'User', foreign_key: 'author_id', counter_cache: :postsCounter
+  has_many :likes
+  has_many :comments
+
+  def recent_comments
+    comments.order(created_at: :desc).limit(5)
+  end
+
```

Comment:
- [ ] [OPTIONAL] **Hint**: *It would be nice to  use one of the [Rails Active Callbacks](https://guides.rubyonrails.org/active_record_callbacks.html) -* `after_save` to call the `#update_post_counter` method every time a new post has been saved to database, then make the `update_post_counter` method.

Reply:
thank you @arslanbisharat but its already auto-updating using `counter_cache`